app/frontend/src/components/Welcome.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <v-container>
+    <div class="mx-auto mb-8 alert info pa-4" style="max-width: 800px;">
+      <h3 style="color: white !important">Attention GETOK users!</h3>
+      <p class="mt-4" style="color: white !important">Access to our hosted services is migrating to the API Services Portal at
+        <a style="color: white !important; text-decoration: underline !important; font-weight: bold" target=" _blank" href="https://api.gov.bc.ca/">api.gov.bc.ca</a>
+        <br>After December 2022, your credentials obtained through GETOK will no longer work.<br>For more guidance on this transition, please read 
+        <a style="color: white !important; text-decoration: underline !important; font-weight: bold" target=" _blank" href="https://github.com/bcgov/common-service-showcase/wiki/CSB-2022-005">more details on our Wiki.</a>
+      </p>
+    </div>
```

Comment:
Maybe not important if we're deprecating this app but I'd really suggest not doing inline style on these.
Also if they're the same banner in both locations it could be a component that's used in 2 spots rather than duplicated

Reply:
My fault.. i suggested this. i know it's horrible.
i was thinking we would roll back this last commit after we take our website down

Reply:
Ah yeah. No prob.
Actually that's a good idea too @TimCsaky . Once we undeploy we could put the request account links back in so the codebase has that functionality should anyone want to ever fork it or reuse it for any other purpose.